src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSetVM/VirtualMachineScaleSetVMUpdateMethod.cs

```diff
@@ -124,4 +124,102 @@ protected PSArgument[] CreateVirtualMachineScaleSetVMUpdateParameters()
                  new object[] { resourceGroupName, vmScaleSetName, instanceId, parameters });
         }
     }
+
+    [Cmdlet(VerbsData.Update, "AzureRmVmssVM", DefaultParameterSetName = "DefaultParameter", SupportsShouldProcess = true)]
+    [OutputType(typeof(PSVirtualMachineScaleSetVM))]
+    public partial class UpdateAzureRmVmssVM : ComputeAutomationBaseCmdlet
+    {
+        public override void ExecuteCmdlet()
+        {
+            ExecuteClientAction(() =>
+            {
+                if (ShouldProcess(this.VMScaleSetName, VerbsData.Update))
+                {
+                    string resourceGroupName;
+                    string vmScaleSetName;
+                    string instanceId;
+                    switch(this.ParameterSetName)
+                    {
+                        case "ResourceIdParameter":
+                            resourceGroupName = GetResourceGroupName(this.ResourceId);
+                            vmScaleSetName = GetResourceName(this.ResourceId, "Microsoft.Compute/VirtualMachineScaleSets", "virtualMachines");
+                            instanceId = GetInstanceId(this.ResourceId, "Microsoft.Compute/VirtualMachineScaleSets", "virtualMachines");
+                            break;
+                        case "ObjectParameter":
+                            resourceGroupName = GetResourceGroupName(this.VirtualMachineScaleSetVM.Id);
+                            vmScaleSetName = GetResourceName(this.VirtualMachineScaleSetVM.Id, "Microsoft.Compute/VirtualMachineScaleSets", "virtualMachines");
+                            instanceId = GetInstanceId(this.VirtualMachineScaleSetVM.Id, "Microsoft.Compute/VirtualMachineScaleSets", "virtualMachines");
+                            break;
+                        default:
+                            resourceGroupName = this.ResourceGroupName;
+                            vmScaleSetName = this.VMScaleSetName;
+                            instanceId = this.InstanceId;
+                            break;
+                    }
+                    VirtualMachineScaleSetVM parameters = new VirtualMachineScaleSetVM();
+                    ComputeAutomationAutoMapperProfile.Mapper.Map<PSVirtualMachineScaleSetVM, VirtualMachineScaleSetVM>(this.VirtualMachineScaleSetVM, parameters);
+
+                    if (this.ParameterSetName != "ObjectParameter")
+                    {
+                        parameters = VirtualMachineScaleSetVMsClient.Get(resourceGroupName, vmScaleSetName, instanceId);
+                    }
+
+                    if (this.DataDisk != null)
+                    {
+                        foreach (var d in this.DataDisk)
+                        {
+                            parameters.StorageProfile.DataDisks.Add(d);
```

Comment:
@hyonholee do we need to do any type of null-check for `StorageProfile` and/or `DataDisks`, or they instantiated when creating the new `VirtualMachineScaleSetVM` object above?

Reply:
@cormacpayne  Thanks, I updated the PR.